src/main/java/com/uipath/uipathpackage/UiPathDeploy.java

```diff
@@ -206,13 +205,22 @@ public void perform(@Nonnull Run<?, ?> run,
         util.validateRuntime(launcher);
 
         try {
-            EnvVars envVars = run.getEnvironment(listener);
+            EnvVars envVars = TaskScopedEnvVarsManager.selectOnlyRequiredEnvironmentVariables(run, env, listener);
+
+            CliDetails cliDetails = util.getCliDetails(run, listener, envVars, launcher);
+            String buildTag = envVars.get("BUILD_TAG");
```

Comment:
"BUILD_TAG" for this, can we use some constants file so that in case any future change is needed, we will just update at that location and the change will get applied at all places?